tests/SprykerTest/Zed/PriceProduct/Business/PriceProductFacadeTest.php

```diff
@@ -47,6 +47,18 @@ class PriceProductFacadeTest extends Unit
     public const EUR_ISO_CODE = 'EUR';
     public const USD_ISO_CODE = 'USD';
 
+    protected const PRICE_TYPE_ORIGINAL = 'ORIGINAL';
```

Comment:
* Undo the changes related to tests from the backport.

Reply:
Done